content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -288,6 +289,10 @@ parameters:
   is specified, volumes are generally round-robin-ed across all active zones
   where Kubernetes cluster has a node. `zone` and `zones` parameters must not
   be used at the same time.
+* `fstype`: `xfs` and `ext4`. Default: `ext4`
+
+If `fstype` is set to `xfs`, you will need xfs support on HostOS
```

Comment:
@grischdian , 

nit: Would you indent line 294 or add a period at end of line 292 and start the second
sentence on line 292 instead of line 294.

Should the second sentence apply to all file system types, instead of just `xfs`?
Possible edit:
The defined filesystem type must be supported by the host operating system.

Information from the API reference:

https://deploy-preview-18391--kubernetes-io-master-staging.netlify.com/docs/reference/generated/kubernetes-api/v1.17/#storageclass-v1-storage-k8s-io

```
Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified. More info: https://kubernetes.io/docs/concepts/storage/volumes#gcepersistentdisk
```